packages/ag-charts-website/plugins/agHtaccessGen.ts

```diff
@@ -24,16 +24,24 @@ export default function createPlugin(options: Options): AstroIntegration {
     return {
         name: 'ag-htaccess-gen',
         hooks: {
-            'astro:build:done': async ({ dir }) => {
+            'astro:build:done': async ({ dir, routes }) => {
                 if (!options.include) {
                     // eslint-disable-next-line no-console
                     console.info('[agHtaccessGen] .htaccess generation disabled, skipping');
                     return;
                 }
 
+                const redirects = routes
+                    .flatMap(({ route, redirect }) =>
```

Comment:
`redirect` doesn't seem to be in the [astro docs](https://docs.astro.build/en/reference/integrations-reference/#routedata-type-reference), but as mentioned offline, it seems to be in the type definition, and does generate as expected on prod build.